velox/functions/sparksql/specialforms/SparkCastHooks.h

```diff
@@ -34,6 +34,10 @@ class SparkCastHooks : public exec::CastHooks {
   /// number of seconds since the epoch (1970-01-01 00:00:00 UTC).
   Expected<Timestamp> castIntToTimestamp(int64_t seconds) const override;
```

Comment:
Do we still need castIntToTimestamp? We can use castNumberToTimestamp<int64_t> instead.

Reply:
And also remove the function castDoubleToTimestamp

Reply:
I forgot it was also implemented in PrestoCastHook, current implement is OK.